src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,99 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
+  .Button--get-firefox,
+  .AMInstallButton {
+    @include respond-to(medium) {
+      align-self: center;
+      margin-left: auto;
+    }
+  }
+
+  .Icon-trophy {
+    margin: margin-end(4px);
```

Comment:
Sorry for not being more specific in my last comments. This should be using a mixin we have to deal with this kind of thing. The syntax is as follows:

```css
.Icon-trophy {
  @include margin-end(4px);
}
```

You can grep for `margin-end` to see some real world examples.